tests/res/mocks/shieldhit_minimal/detect.dat

```diff
@@ -0,0 +1,14 @@
+Geometry Cyl
+    Name ScoringCylinder
+        R 0.0 10.0 4
+        Z 0.0 30.0 4
```

Comment:
Better to use cartesian mesh divived into 4x4 bins, dividing cylinder along R and along Z is quite rare.